packages/core/docs/migrate/2.3.0-rc.1/overview.md

```diff
@@ -0,0 +1,151 @@
+      }
+    }
+  }
+};
+
+// Some component
+const { search } = useProduct();
+
+onSSR(async () => {
+  await search({ customQuery: { products: 'MyCustomQuery' } })
+});
+```
+
+As you can in the example above, `customQuery` is now an object that accepts object with:
+- key matching the name of GraphQL query to override,
+- value matching the name of the handler in `middleware.config.js`.
+
+## Checkout
+
+One of the most important pieces missing in VSF Next was Checkout, which originally was supposed to be handled by a single `useCheckout` composable. However, we decided that this approach won't scale well because the checkout process is very different in every integration and it might not work well, if multiple providers will be used to handle different steps of the checkout process, like payment, shipping, etc.
+
+### Changes
+
+We split responsibilities of different steps into multiple new composables:
+
+- `useShipping` for handling shipping information,
+- `useBilling` for handling billing information,
+- `useShippingProvider` for handling shipping providers,
+- `useMakeOrder` for placing the final order.
+
+We also used this opportunity to cleanup other composables:
+- removed `useCheckout`,
+- removed `checkoutGetters`,
+- renamed `useUserOrders` to `useUserOrder` to be consistent with other composables,
+- renamed `isOnCart` and `isOnWishlist` in `useCart` composable to `isInCart` and `isInWishlist`.
+
+These changes affected multiple Vue components. Please refer to [UI section](#ui) for more information.
+
+## Cache
+
+TODO:
+- update Cache library
+
+## UI
+
+We added and updated multiple UI elements:
+
+- renamed `components/checkout` directory in core theme to `components/Checkout` (note the capital `C`). Please update your imports.
+- updated `components/AppFooter.vue`,
+- updated `components/AppHeader.vue`,
+- updated `components/BottomNavigation.vue`,
+- updated `components/Checkout/CartSidebar.vue`,
+- updated `components/InstagramFeed.vue`,
+- updated `components/LocaleSelector.vue`,
+- updated `components/MobileStoreBanner.vue`,
+- updated `components/TopBar.vue`,
+- updated `components/WishlistSidebar.vue`,
+- updated `layouts/default.vue`,
+- updated `layouts/error.vue`,
+- updated `pages/Category.vue`,
+- updated `pages/Checkout/OrderReview.vue`,
+- updated `pages/Home.vue`,
+- updated `pages/MyAccount.vue`,
+- updated `pages/MyAccount/OrderHistory.vue`,
+- updated `pages/Product.vue`,
+- updated `composables/useUiHelpers/index.ts`:
+  - renamed `changeSearchTerm` to `setTermForUrl`,
+  - added `getSearchTermFromUrl`,
+- updated translations in `theme/lang`,
+- added new component `components/Notification.vue`,
+- added new component `components/SearchResults.vue`,
+- added new, integration-specific middleware `middleware/is-authenticated.js`,
+- added new integration-specific component `components/Checkout/CartPreview.vue` (previously implemented as `components/checkout/CartPreview.vue` in the core theme).
```

Comment:
Maybe `added new integration-specific component` to `added new, integration-specific component` as in line above?